ocaml/xapi/xapi_vdi.ml

```diff
@@ -625,13 +625,27 @@ let create ~__context ~name_label ~name_description ~sR ~virtual_size ~_type
     | `cbt_metadata ->
         "cbt_metadata"
   in
+  (* special case: we want to use a specific UUID for Pool Meta Data
+     Backup *)
+  let uuid_ =
+    match (_type, name_label) with
+    | `user, "Pool Metadata Backup" ->
```

Comment:
Will the VDI type and name label still be preserved in the scenario we want to use this for, i.e. recovery of a lost XAPI database?
I think all this info would've been stored in the xapi database itself, the only thing you can know about the VDI is its UUID, no other metadata would survive a loss of the XAPI database.

Reply:
Would be good to have a test for this (obviously not here, but in XenRT) that tries to exercise the scenario where this backup/recovery is meant to be used and check that you can still recover starting from a fresh install, and attaching an SR.